### ISSUE TITLE
Touch Controller Support

### ISSUE THREAD

**Issue by shadowbrony33** [CarlKenner/dolphin#119]
Referencing some of #111 

> Despite this being an old issue, it would be best to implement proper support for the Oculus Touch controllers due to its market share of ~46% (According to Steam HW Survey of December 2017).
>Please add support for the people out there (myself included) who don't have anything else but the touch controllers for VR.

Just pushing this to an open issue.

**Comment by Kosyne** [comment]
Yeah, same. I hear tell that the main dolphin team may give it a shot when OpenXR comes out, but I like actually being able to play using the oculus SDK, so I think this version may have the edge in that regard, just needs updates.

**Comment by CarlKenner** [comment]
I decided to abandon my unsuccessful boycott of adding Touch controller support, so I will do this now.
It shouldn't be too hard, I already did some of the work on the textures ages ago, and I just need to use the code I already wrote for the Vive controllers.

**Comment by shadowbrony33** [comment]
@CarlKenner Thank you so much!

**Comment by 1kozlovsas** [comment]
Looking forward to this enhancement! If there is any way I can assist you in debugging/testing please let me know.
